edgetpu_node_manager:
  nodes:
    - name: edgetpu_face_detector
      type: face_detector
    - name: edgetpu_object_detector
      type: object_detector
    - name: edgetpu_human_pose_estimator
      type: human_pose_estimator
    - name: edgetpu_semantic_segmenter
      type: semantic_segmenter
  default: edgetpu_human_pose_estimator
edgetpu_face_detector:
  model_file: package://coral_usb/models/mobilenet_ssd_v2_face_quant_postprocess_edgetpu.tflite
  image_transport: raw
  device_id: 0
edgetpu_object_detector:
  model_file: package://coral_usb/models/mobilenet_ssd_v2_coco_quant_postprocess_edgetpu.tflite
  label_file: package://coral_usb/models/coco_labels.txt
  image_transport: raw
  device_id: 0
edgetpu_human_pose_estimator:
  model_file: package://coral_usb/python/coral_usb/posenet/models/mobilenet/posenet_mobilenet_v1_075_481_641_quant_decoder_edgetpu.tflite
  image_transport: raw
  device_id: 0
edgetpu_semantic_segmenter:
  image_transport: raw
  device_id: 0
